src/sentry/event_manager.py

```diff
@@ -1790,6 +1792,8 @@ def _handle_regression(group: Group, event: Event, release: Optional[Release]) -
 
     group.active_at = date
     group.status = GroupStatus.UNRESOLVED
+    group.substatus = GroupSubStatus.REGRESSED
+    group.save()
```

Comment:
I dont think this is needed here since its already updated at line 1776.

Reply:
I think we should set the substatus explicitly whenever we set the group status, wdyt?

Reply:
Setting the substatus explicitly makes sense since its used downstream, but I don't believe the `save()` part is needed. Unless I'm missing something.